vendor/assets/stylesheets/rolodex/components/_labels.sass

```diff
@@ -2,17 +2,17 @@
 // ========================================
 
 .label
-  @extend %background-white
-  @extend %border-radius-mini
-  @extend %border-solid
-  @extend %border-1
+  background-color: $white
+  +rem(border-radius, $border-radius-mini)
+  border-style: solid
+  +rem(border-width, 1px)
+  +s-hotel
   font-weight: 500
```

Comment:
Labels use Helvetica, so `500` is a semibold weight. It's the only place we use Helvetica in semibold thus the only place it appears.